docs/api/utils.md

```diff
@@ -76,6 +76,8 @@ await Apify.utils.enqueueLinks({
 
     -   **`[.page]`**: `Page` - Puppeteer [`Page`](https://pptr.dev/#?product=Puppeteer&show=api-class-page) object. Either `page` or `$` option must
         be provided.
+    -   **`[.limit]`**: `Number` - Limit the count of actually enqueued URLs to this number. Useful for testing across the entire crawling scope.
+        be provided.
```

Comment:
Docs are autogenerated, please update the JSDoc comment.

Reply:
Right... will fix